### ISSUE TITLE
Enhancement:  to (open) Windows dialog -- add sortable "file changed" column

### ISSUE THREAD

**Issue by HerbM** [notepad-plus-plus/notepad-plus-plus#8456]
### Description of the Issue
Enhancement:  to (open) Windows dialog -- **add sortable "file changed" column**

### Steps to Reproduce the Issue
Open Windows dialog (Alt-w-w)
File change indiciation is appended to file name (which is fine) but doesn't appear in separate sortable column

### Expected Behavior
**Add column for "changed" indicator so it will be sortable by "changed status"**

### Actual Behavior
File change indiciation is appended to file name (which is fine) but doesn't appear in separate sortable column

### Debug Information
[This is an enhancement request so have nothing to do with current versions]

Notepad++ v7.8.5   (64-bit)
Build time : Mar  4 2020 - 11:08:34
Path : C:\Util\notepad++.exe
Admin mode : ON
Local Conf mode : OFF
OS Name : Windows 10 Pro (64-bit) 
OS Version : 2004
OS Build : 19035.1000
Plugins : BetterMultiSelection.dll ComparePlugin.dll CSScriptNpp.dll CsvQuery.dll DSpellCheck.dll JSMinNPP.dll LuaScript.dll MarkdownViewerPlusPlus.dll mimeTools.dll NppConverter.dll nppcrypt.dll NppEventExec.dll NppExec.dll NppExport.dll NppRegExTractorPlugin.dll NppSnippets.dll NppTaskList.dll NppToolBucket.dll pork2sausage.dll SecurePad.dll SelectQuotedText.dll SurroundSelection.dll XBrackets.dll XMLTools.dll _CustomizeToolbar.dll

**Comment by sasumner** [comment]
>This is an enhancement request so have nothing to do with current versions

I would disagree.  If you're going to cite behavior that you'd like to see changed, it's good to confirm what version you're basing your change request on.
